client/components/place/place-list/place-list.component.js

```diff
@@ -107,6 +111,7 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         ctrl.hidePopularTracks = true;
 
         ctrl.checkPopularPlaces = function() {
+          angular.element('.popular-places-wrapper').css('display', 'block');
```

Comment:
Because, by default it's display: none. And after activation button it should be display: block. And at this moment fires ng-hide with ng-animate